app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
Yeah, I use it for the top level `Settings`.  Code wise it's as simple as: `Config::Options.new(hash)`.  However you can't call `.reload!` on that because it doesn't have any "sources".